packages/sdk/src/machines/purchase-initiation-machine/index.ts

```diff
@@ -451,6 +489,18 @@ export const purchaseInitiationMachine = setup({
         },
       },
     },
+    waitingForStart: {
+      always: {
+        target: "readyForPurchase",
+        guard: "isSaleStarted",
+      },
+      after: {
+        1000: {
```

Comment:
Just to double, this shouldn't trigger any api or web3 calls, right? that's what it looks like to me, but checking since this is 1sec

Reply:
yup! it just pulls the live sale data from the parent project sale machine which is updated with new data every 10 seconds.

Reply:
and just web3 calls